developer/services/disposablevm-implementation.rst

```diff
@@ -2,73 +2,162 @@
 Disposable implementation
 =========================
 
+.. warning::
 
-**Note: The content below applies to Qubes R3.2.**
+      This page is intended for advanced users.
 
-DisposableVM image preparation
-------------------------------
+Disposable behavior
+-------------------
 
+A :term:`disposable template` is a qube which has the :py:class:`AppVM <core-admin:qubes.vm.appvm.AppVM>` class and the :py:attr:`template_for_dispvms <core-admin:qubes.vm.mix.dvmtemplate.DVMTemplateMixin.template_for_dispvms>` property enabled, being a :py:class:`DVMTemplateMixin <core-admin:qubes.vm.mix.dvmtemplate.DVMTemplateMixin>`.
```

Comment:
Here, but also in several parts later: it's better to first say "why", and only then "how", otherwise reading the "how" part lacks context. Here it would be something like:
> A "disposable template" is a template holding all configuration parameters to create new disposables. It's a qube with "template_for_dispvms" property enabled, can be either AppVM or Standalone.